src/metabase/collections/api.clj

```diff
@@ -980,12 +981,6 @@
                   :can_restore
                   :can_delete)))
 
-(api.macros/defendpoint :get "/:id"
-  "Fetch a specific Collection with standard details added"
-  [{:keys [id]} :- [:map
-                    [:id ms/PositiveInt]]]
-  (collection-detail (api/read-check :model/Collection id)))
```

Comment:
I need to move this below, otherwise `:get /trash`, `:get /personal` and `:get /root` won't be called as it reaches `get /:id` which now accepts a string (entity id) first.